dotCMS/src/main/java/com/dotcms/rest/api/v1/folder/FolderResource.java

```diff
@@ -91,7 +91,11 @@ public final Response loadFolderByURI(@Context final HttpServletRequest httpServ
         final InitDataObject initData = this.webResource.init(null, httpServletRequest, httpServletResponse, true, null);
         final User user = initData.getUser();
         try{
-            Folder folder = folderHelper.loadFolderByURI(siteName,user,uri);
+
+            String uriParam = uri;
+            if(!uriParam.startsWith("/")) uriParam = "/".concat(uriParam);
```

Comment:
Codacy found an issue: [This call to String.startsWith can be rewritten using String.charAt(0)](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5795082)